src/components/organisms/topbar/project/buttons.tsx

```diff
@@ -158,6 +158,43 @@ export const ProjectTopbarButtons = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	const exportProject = useCallback(async () => {
+		const { data: akProjectArchiveZip, error } = await ProjectsService.export(projectId!);
+
```

Comment:
add state isExporting and `Loader` for Button to show that something is happening

Reply:
done